code/src/components/Summary.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react';
+import Button from './Button';
+
+const Summary = ({
+  data,
+  totalQuestions,
+  replyCourse,
+  replyExperience,
+  replyFavourite,
+  replyImprove,
+  replyImproveDetails,
+  replyRating,
+  handleReset
+}) => {
+  return (
+    <div id={`card-${totalQuestions}`} className="summary-wrapper inactive">
+      <h1>Summary</h1>
+      <div className="card card-summary">
+        <p className="summary-question summary-question-1">
+          {data.questions[0].question_text}
+        </p>
+        <p className="summary-reply summary-reply-1">
+          {typeof replyCourse !== 'object' ? replyCourse : '-'}
```

Comment:
And we are back to why initialising our state properties is better when we stick to same data type ;)
If you would initialise them as empty strings (or at least most of them), over here you could instead write :

```
{replyCourse || '-'}
```